src/components/button/Button.js

```diff
@@ -1,22 +1,23 @@
 import React, { useState } from 'react'
 
-const Button = () => {
+const Button = ({variant, size, text}) => {
   const [loader, setLoader] = useState(false)
-  const hanleClick = () => {
+  const [disable, setDisable] = useState(true)
+  const handleClick = () => {
     setLoader(true)
```

Comment:
this would mean anytime button is clicked, it will call loader to be true..
what if i want a different use case?
that should be controlled outside the component with the loader prop